typed-racket-doc/typed-racket/scribblings/reference/types.scrbl

```diff
@@ -354,6 +354,12 @@ corresponding to @racket[trest], where @racket[bound]
 
 @ex[(box "hello world")]
 
+@defform[(Read-Only-Boxof t)]{A read-only @rtech{box} of @racket[t].}
+
```

Comment:
I agree that explaining read-only boxes in terms of `const` would be confusing. I'm suggesting to just mention it, as in "if you know `const` from C, this is similar".